source/dotnet/Library/AdaptiveCards/AdaptiveBaseElement.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Xml.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace AdaptiveCards
+{
+    public class AdaptiveInternalID
```

Comment:
we should maybe start having xml style comments or descriptions on our classes/functions etc at some point for readability/maintainability

Reply:
Agreed. I've filed #2745 to track this.

---
In reply to: [278314967](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278314967) [](ancestors = 278314967)